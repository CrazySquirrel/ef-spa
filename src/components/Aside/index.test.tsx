import * as React from 'react';
import {shallow} from 'enzyme';

import {Aside} from './index';

describe('Aside', () => {
  it('render', () => {
    const result = shallow(<Aside/>);

    expect(result).toMatchSnapshot();
  });
});
