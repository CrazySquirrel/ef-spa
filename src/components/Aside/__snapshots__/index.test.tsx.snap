version https://git-lfs.github.com/spec/v1
oid sha256:04cd22e3ffc421beb380b9db580f914879f36015873d76ca5cb2e3cec3f03020
size 2532
